smart_campus/app/urls.py

```diff
@@ -15,5 +15,6 @@
     url('^update_user_experience_point/$', views.update_user_experience_point, name='Update user experience point'),
     url('^add_user_favorite_stations/$', views.add_user_favorite_stations, name='Add user favorite stations'),
     url('^remove_user_favorite_stations/$', views.remove_user_favorite_stations, name='Remove user favorite stations'),
-    url('^get_all_travel_plans/$', views.get_all_travel_plans, name='Get all travel plans')
+    url('^get_all_travel_plans/$', views.get_all_travel_plans, name='Get all travel plans'),
+    url('^add_user_reward/$', views.add_user_reward, name="Add User Reward")
```

Comment:
'Add user reward'
Also, it would be better if you unify your style of using `'` and  `"` for a string except for some exceptions (e.g. "don't")

Reply:
but the style of urls in the smart_campus is 'Add User Reward

Reply:
You should.

Reply:
Okay, unified it.